emoji/src/main/java/com/vanniktech/emoji/EmojiImageView.java

```diff
@@ -122,11 +122,26 @@ public boolean onLongClick(final View view) {
     }
   }
 
-  public void setOnEmojiClickListener(@Nullable final OnEmojiClickListener listener) {
+  /**
+   * Updates the emoji image directly. This should be called only for updating the variant
+   * displayed (of the same base emoji), since it does not run asynchronously and does not update
+   * the internal listeners.
+   *
+   * @param emoji The new emoji variant to show.
+   */
+  void updateEmoji(@NonNull final Emoji emoji) {
```

Comment:
should this be public? otherwise you can't access it when you want to do your custom implementation